framework/contracts/native/account-factory/src/commands.rs

```diff
@@ -376,3 +390,25 @@ pub fn execute_update_config(
 
     Ok(AccountFactoryResponse::action("update_config"))
 }
+
+pub(crate) fn verify_nft_ownership(
```

Comment:
This function is duplicated with the manager contract, could you port that inside abstract-std instead ? Inside objects/gov_type for instance